src/science/radiation/cbl_init_radiation.F90

```diff
@@ -378,9 +378,8 @@ SUBROUTINE BeamFraction( RadFbeam, mp, nrb, Cpi,Ccoszen_tols_huge, metDoy,     &
 
 
 ! Define beam fraction, fbeam:
-! #355 beam fraction defined using total SW_down; applies to VIS&NIR equally
-RadFbeam(:,1) = spitter(mp, cpi, metDoy, coszen, SW_down(:,1)+SW_down(:,2))
-RadFbeam(:,2) = RadFbeam(:,1)
+RadFbeam(:,1) = spitter(mp, cpi, metDoy, coszen, SW_down(:,1))
+RadfBeam(:,2) = spitter(mp, cpi, metDoy, coszen, SW_down(:,2))
```

Comment:
Please check with @rml599gh - I think this is reinstating a bug that we identified in CABLE3-ESM1.6 development and so the change needs to go the other way.

Reply:
Yes, #355, #354 describe this bug. Code in red is the correct version. @har917 - great that you picked this up. Highlights the value of reviews.

Reply:
fixed, sorry - IDK how this happened